### PR TITLE
Add OAreport, OAsupport

### PR DESCRIPTION
**Pull request by sssoz**

### CODE REVIEW

**Comment by sssoz** [comment]
@JosephMcArthur Still a WIP but I’ve added you as a reviewer if you wanted to check out the copy bits I added first.

File: src/_data/products.json [hunk 1]
```diff
@@ -25,14 +25,36 @@
...
     "featured": true
+  },
+  {
+    "logo": "",
+    "name": "OAreport",
+    "url": "https://oa.report",
+    "label": "Open Access reporting",
+    "tagline": "Put OA policies into practice in minutes, not months.",
+    "cta": "Sign up for updates",
+    "description": "OAreport translates policy to impact by helping libraries and funder discover papers covered by your policy, analyze if and how papers comply, and unlock individual papers and systemic changes.",
```
Comment (JosephMcArthur) [inline src/_data/products.json:45]: "your policy" → "their policy"

File: src/_data/products.json [hunk 2]
```diff
@@ -25,14 +25,36 @@
   },
   {
     "logo": "products/oab.svg",
-    "name": "OAButton",
+    "name": "OAbutton",
```
Comment (JosephMcArthur) [inline src/_data/products.json:28]: Good catch

File: src/_data/products.json (hunk 1 above)
Comment (sssoz) [reply src/_data/products.json:45]: That’s what on the OA.report website -- should we also update it there then?

Comment (JosephMcArthur) [reply src/_data/products.json:45]: Hmm, for some reason on those sites it feels like it's talking to "you", where as here it feels more like "them" –– so I think the change makes sense here but not there.

However, I generally feel like you're better at these sorts of things than me. So, if you disagree feel free to go with your preference.

Comment (sssoz) [reply src/_data/products.json:45]: Well the correct way would be "their policy" of course (as suggested here). What’s on the OAreport website is grammatically incorrect.

**Comment by JosephMcArthur** [comment]
Then let's go with their in both places. Thanks again!

Joseph McArthur
Assistant Director: Right to Research Coalition <http://righttoresearch.org>
Director / Founder: Open Access Button <http://openaccessbutton.org>
Twitter: @Mcarthur_Joe <https://twitter.com/Mcarthur_Joe>
Skype: joseph_mcarthur

File: src/_data/products.json (hunk 1 above)
Comment (sssoz) [reply src/_data/products.json:45]: @JosephMcArthur  "Then let's go with their in both places. Thanks again!"

Comment (sssoz) [reply src/_data/products.json:45]: Corrected to "by their policies"

**Pull request merged**